tests/kafkatest/services/security/security_config.py

```diff
@@ -350,6 +362,14 @@ def kafka_opts(self):
         else:
             return ""
 
+    def export_kafka_opts_for_admin_client_as_broker(self):
```

Comment:
As we discussed offline, I think eventually the admin client should log in as its own admin user, not as the broker.  But we can save that for a follow-on.

Reply:
Yes, created https://issues.apache.org/jira/browse/KAFKA-10443 to track this.